server/sonar-web/src/main/js/components/controls/SearchSelect.tsx

```diff
@@ -73,11 +77,15 @@ export default class SearchSelect extends React.PureComponent<Props, State> {
     return this.props.resetOnBlur !== undefined ? this.props.resetOnBlur : true;
   }
 
-  handleSearch = (query: string) =>
-    this.props.onSearch(query).then(
+  handleSearch = (query: string) => {
+    const copyQuery = query;
```

Comment:
I'm not sure I understand what you're doing here

Reply:
It ignores the result if the query changed.